sw/host/opentitantool/src/command/gpio.rs

```diff
@@ -75,11 +75,11 @@ pub struct GpioSetDirection {
     pub pin: String,
     #[structopt(
         name = "DIRECTION",
```

Comment:
We can change this to `MODE`.

Reply:
I have also renamed the command from GpioSetDirection to GpioSetMode, and added a separate command to control the weak pull mode.